### PR TITLE
Seller show

### PR DESCRIPTION
**Pull request by MaxMillington**
Made is so that there are stupid robot photos on the sellers index page, and working links that take you to the sellers show page that will show you active auctions with links to the individual auction.

### CODE REVIEW

**Comment by sabondano** [comment]
:+1:

File: app/views/auctions/index.html.erb [hunk 1]
```diff
@@ -56,16 +56,16 @@
...
-        <!--</li>-->
-      <!--</ul>-->
-    <!--</div>-->
-  <!--</div>-->
+   <!--Pagination-->
+  <div class="row text-center">
+    <div class="col-lg-12">
+      <ul class="pagination">
+        <li>
+          <a href="<%#= will_paginate @auctions %>"></a>
```
Comment (sabondano) [inline app/views/auctions/index.html.erb]: @MaxMillington do we want to leave this line as a comment?

**Comment by MaxMillington** [comment]
No, we should have active pagination. I removed the comment. It should be good to go now.

**Comment by sabondano** [comment]
:+1:

**Pull request merged**